app/controllers/users_controller.rb

```diff
@@ -1,4 +1,5 @@
 class UsersController < ApplicationController
+  layout 'my_orders', only: [:orders, :line_items]
```

Comment:
use symbol

Reply:
If we want to define a specific layout we have to pass string only, if we pass symbol, it searches for any variable or function not the layout